cves/CVE-2014-0117.yml

```diff
@@ -95,9 +105,14 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer:
-  code:
-  fix:
+  answer: |
```

Comment:
So after the fix was committed, did they do something else to test to see if the fix was correct?